### PR TITLE
HPCC-13351 FLZ compress/decompress fix

### PR DESCRIPTION
**Pull request by mckellyln**
Signed-off-by: Mark Kelly mark.kelly@lexisnexis.com

### CODE REVIEW

**Comment by mckellyln** [comment]
@jakesmith or @ghalliday for your review.

**Comment by ghalliday** [comment]
@jakesmith please double check.  Also, should this be going in 5.2.2 rather than 5.4.  It is fixing a bug, and probably fairly significant.

File: system/jlib/jlzw.cpp [hunk 1]
```diff
@@ -2268,6 +2268,8 @@ ICompressedFileIO *createCompressedFileWriter(IFileIO *fileio,size32_t recordsiz
         trailer.crc = ~0U;
         trailer.compressedType = fast?FASTCOMPRESSEDFILEFLAG:COMPRESSEDFILEFLAG;
         trailer.blockSize = COMPRESSEDFILEBLOCKSIZE;
+        if (fast)
+            trailer.blockSize *= 4;
```
Comment (jakesmith) [inline system/jlib/jlzw.cpp]: A comment would be useful here as to why fast needs/is better with * 4 block size, vs non-fast not.

**Comment by jakesmith** [comment]
Looks okay to me, one minor comment.
@mckellyln

**Comment by mckellyln** [comment]
Addressing issue in HPCC-9920 PR

**Pull request closed without merging**